.github/workflows/cocoapods.yml

```diff
@@ -0,0 +1,29 @@
+name: Cocoapods
+
+on:
+  push:
+    branches: stable
+    paths:
+      - '.github/workflows/cocoapods.yml'
+      - '*.podspec'
+  pull_request:
+    paths:
+      - '.github/workflows/cocoapods.yml'
+      - '*.podspec'
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - 
+        name: Checkout
+        uses: actions/checkout@v2
```

Comment:
nit: why the odd format of leaving the first line blank?

Reply:
Dunno, saw this format in some examples and other docs, and looks a bit clearer to me in editor 🤷

Reply:
Fair enough